web-animations-1/Overview.bs

```diff
@@ -2344,19 +2404,27 @@ definitions:
 
 An <a>animation effect</a> is in the <dfn>before phase</dfn> if the
 animation effect's <a>local time</a> is not <a>unresolved</a> and
-<em>either</em> of the following conditions are met:
+<em>any</em> of the following conditions are met:
 
 1.  the <a>local time</a> is less than the <a>before-active boundary time</a>,
     <em>or</em>
+1.  the effect is <a>associated with an animation</a> <em>and</em> the
+    associated <a>animation</a>'s <a>current phase</a> is
+    [=timeline before phase|before=] <em>and</em> the <a>local time</a> is equal
+    the <a>before-active boundary time</a>, <em>or</em>
 1.  the <a>animation direction</a> is &lsquo;backwards&rsquo; and the <a>local
     time</a> is equal to the <a>before-active boundary time</a>.
 
 An <a>animation effect</a> is in the <dfn>after phase</dfn> if the
 animation effect's <a>local time</a> is not <a>unresolved</a> and
-<em>either</em> of the following conditions are met:
+<em>any</em> of the following conditions are met:
 
 1.  the <a>local time</a> is greater than the <a>active-after boundary
     time</a>, <em>or</em>
+1.  the effect is <a>associated with an animation</a> <em>and</em> the
+    associated <a>animation</a>'s <a>current phase</a> is
+    [=timeline after phase|after=] <em>and</em> the <a>local time</a> is equal
+    to the <a>active-after boundary time</a>, <em>or</em>uirecjkeiucfjseuejdjijauzsuwqaiuaizjjsujzisaijreuwnghweww
```

Comment:
The "uirecjkeiucfjseuejdjijauzsuwqaiuaizjjsujzisaijreuwnghweww" part here is interesting but probably not necessary 😅

Reply:
Surprise benefits of working from home :) My 3 year old daughter decided to try and do some of my work lol. I usually catch those, she was extra sneaky this time.